tests/unit/CountItems/CountWorker.js

```diff
@@ -213,15 +205,52 @@ describe('CountItems::CountWorker', () => {
             _creationDate: Date.now().toString(),
             website: { indexDocument: 'index.html' },
         };
+        w.getIsTransient = jest.fn((bucketInfo, cb) => cb(null, true));
         mongoMock.setup.mockImplementationOnce(cb => cb());
         mongoMock.close.mockImplementationOnce(cb => cb());
         mongoMock.client.isConnected.mockImplementationOnce(() => false);
-        mongoMock._getIsTransient.mockImplementationOnce((_a, _b, cb) => cb(null, true));
         mongoMock.getObjectMDStats.mockImplementationOnce((_a, _b, _c, _d, cb) => cb(null, { value: 42 }));
         w.countItems(bucketInfo, (err, results) => {
             expect(err).toBeNull();
             expect(results).toEqual({ value: 42 });
             done();
         });
     });
+
+    test('should use real getIsTransient and not crash', done => {
```

Comment:
should have at least 2 scenarios, depending on `this.client.isLocationTransient` (since there are multiple code path).

the point is not just to see that `countItems` is calling `getIsTransient`, but really to execute the code in `getIsTransient` to ensure it will not fail in production... (it may be done though other tests, calling the function directly!)